### PR TITLE
Table overview apiv3

### PR DESCRIPTION
**Pull request by Jonasvsl**
Please check the table that was added/changed on the status of implementation of API v3.

This table is meant to give users a clear overview of how they can use 3Di

Suggestions for a different/better location in the docs are welcome

### CODE REVIEW

File: source/d_qgis_plugin.rst [hunk 1]
```diff
@@ -81,15 +81,24 @@ Simulate
...
-Forcings                        Spatialite          API             Live site
-============================= ================= ================= ===============
-Boundary conditions             v1, v3              v3              v1, v3
-Initial water level             v1, v3              v1, v3          v3
-Laterals  1D and 2D             v1                  v3              -
-DWF (inflow)                    v1                  v3              -
-============================= ================= ================= ===============
-
+============================= =========================== ===================================== ==================================
+Forcings                        Live site                  3Di API Client                       OpenAPI client
```
Comment (nvolp) [inline source/d_qgis_plugin.rst:85]: 3Di API Client Wizard

File: source/d_qgis_plugin.rst [hunk 2]
```diff
@@ -81,15 +81,24 @@ Simulate
...
+============================= =========================== ===================================== ==================================
+Boundary conditions            SQLite                     SQLite                                SQLite, can be overwritten
```
Comment (nvolp) [inline source/d_qgis_plugin.rst:87]: OpenAPI Client, SqLite can be overwritten, but in that case all boundary conditions (both 1D and 2D) need to be overwritten

File: source/d_qgis_plugin.rst [hunk 3]
```diff
@@ -81,15 +81,24 @@ Simulate
...
+Initial water level 2D         SQLite, always 'max'       Add raster/global in wizard           Add raster/global to simulation
```
Comment (nvolp) [inline source/d_qgis_plugin.rst:88]: Via de API Client & OpenAPI client: max, min, avg can be chosen, raster can be chosen, but raster must have been uploaded

File: source/d_qgis_plugin.rst [hunk 4]
```diff
@@ -81,15 +81,24 @@ Simulate
...
+Initial water level 1D         SQLite                     Add predefined in wizard              Add predefined to simulation
```
Comment (nvolp) [inline source/d_qgis_plugin.rst:89]: Wizard and OpenAPI, from SqLite or constant value

File: source/d_qgis_plugin.rst (hunk 3 above)
Comment (nvolp) [reply source/d_qgis_plugin.rst:88]: Ik mis grondwater

File: source/d_qgis_plugin.rst [hunk 5]
```diff
@@ -81,15 +81,24 @@ Simulate
...
+Laterals  1D and 2D            Not used                   Add in wizard with CSV*               Add CSV
```
Comment (nvolp) [inline source/d_qgis_plugin.rst:90]: Single timeseries are also possible up to 300 entries

File: source/d_qgis_plugin.rst [hunk 6]
```diff
@@ -81,15 +81,24 @@ Simulate
...
+Breaches			           Open breach on live site   Open breach using wizard              Open breach
```
Comment (nvolp) [inline source/d_qgis_plugin.rst:91]: uitlijnen klopt niet

File: source/d_qgis_plugin.rst (hunk 1 above)
Comment (Jonasvsl) [reply source/d_qgis_plugin.rst:85]: changed

File: source/d_qgis_plugin.rst (hunk 2 above)
Comment (Jonasvsl) [reply source/d_qgis_plugin.rst:87]: added under the table

File: source/d_qgis_plugin.rst (hunk 3 above)
Comment (Jonasvsl) [reply source/d_qgis_plugin.rst:88]: gw toegevoegd in tabel

**Pull request merged**